src/main/java/edu/greenblitz/tobyDetermined/subsystems/telescopicArm/Elbow/Elbow.java

```diff
@@ -33,6 +30,10 @@ public class Elbow extends GBSubsystem {
 
     private boolean debug = false;
 
+
+    private IElbow io;
```

Comment:
Should not be named io, this is the elbow and should be named liked it (also in other cases, i.e. gyro, belly,...)